test.sh

```diff
@@ -9,7 +9,11 @@ export PDFium_DIR="$PWD/staging"
 
 mkdir build
 cd build
-cmake "$SOURCE_DIR"
+if [ "$TARGET_CPU" == "arm" ]; then
```

Comment:
I have very limited experience with ARM build, so let me know if I'm wrong.
It seems to me that `arm-linux-gnueabihf-gcc` is not the only compiler that can compile for ARM and that this name is incidental to the package we installed from the Travis configuration.
Therefore, I think we should try to get this name out of the build script to make if more versatile.

We could make something more generic based on the `_CC` and `_CXX` variables you defined in the YAML configuration.
Indeed, [CMake will automatically use the compilers defined in the `CC` and `CXX` environment variables](https://cmake.org/cmake/help/latest/envvar/CC.html), so you could remove this entire block and, instead, define these variables at the top of the script, like so (not tested):

```bash
# Travis overrides CC and CXX, so we had to use different names
[ -n "$_CC" ] && export CC="$_CC"
[ -n "$_CXX" ] && export CC="$_CXX"
```

Alternatively, you could place these lines in the YAML file, before calling the build script; this way it would keep all the Travis specific code in the right place.

Reply:
Ah, nice. I added the lines to the top of the file.